<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rincon Denver GTOCX Visualizer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Time Galaxy Model</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="GLViewWidget" name="time_graphics_view"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Time</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="time_click_box">
              <property name="singleStep">
               <double>0.100000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="time_slider">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GLViewWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph.opengl</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>time_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>update_time_from_slider(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>539</y>
    </hint>
    <hint type="destinationlabel">
     <x>45</x>
     <y>575</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>time_click_box</sender>
   <signal>valueChanged(double)</signal>
   <receiver>MainWindow</receiver>
   <slot>update_time_from_click_box(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>111</x>
     <y>546</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>575</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>update_time_from_click_box(double)</slot>
  <slot>update_time_from_slider(int)</slot>
 </slots>
</ui>
